src/lib/utils.ts

```diff
@@ -1,18 +1,304 @@
-import { Context } from "telegraf";
-import { EBotUserState, IBotUser } from "../interfaces/bot-users";
+import { Context } from 'telegraf';
+import { EBotUserState, IBotUser } from '../interfaces/botUser';
+import axios from 'axios';
+import { v4 as uuidv4 } from 'uuid';
+import { IChompUser, IChompUserResponse } from '../interfaces/chompUser';
+import { IQuestion } from '../interfaces/question';
+import { IAnswerResponse } from '../interfaces/answer';
+
+const BOT_API_KEY = process.env.BOT_API_KEY;
+const WEB_APP_URL = process.env.WEB_APP_URL;
+const DYNAMIC_TOKEN = process.env.DYNAMIC_TOKEN;
+const DYNAMIC_ENVIRONMENT_ID = process.env.DYNAMIC_ENVIRONMENT_ID;
+
+const headers = {
+  Authorization: `Bearer ${DYNAMIC_TOKEN}`,
+  'Content-Type': 'application/json',
+};
+const baseUrl = `https://app.dynamicauth.com/api/v0`;
 
 export const adaptCtx2User = (ctx: Context) => {
-    const newUser: IBotUser = {
-        tg_id: ctx.from?.id!,
-        tg_first_name: ctx.from?.first_name!,
-        tg_username: ctx.from?.username!,
-        tg_is_bot: ctx.from?.is_bot!,
-        tg_language_code: ctx.from?.language_code!,
-        state: EBotUserState.NEW,
-        original_email_address: "overwrite",
-        // TODO use chomp API to get an ID
-        // chomp_id: "TODO",
+  const newUser: IBotUser = {
+    tg_id: ctx.from?.id!,
+    tg_first_name: ctx.from?.first_name!,
+    tg_username: ctx.from?.username!,
+    tg_is_bot: ctx.from?.is_bot!,
+    tg_language_code: ctx.from?.language_code!,
+    state: EBotUserState.NEW,
+    original_email_address: 'overwrite',
```

Comment:
wouldn't it make more sense to make this an optional field with no default value?